brianlong/www.validators.app
# 186711902 api for skipped after

#### What's this PR do?
- add skipped after percent to api and docs

#### How should this be manually tested?
- run tests
- run gather and score daemons
- check validator index and details api - they should both display correct skipped_after_percent values
- check if response times are not too long
- check if the value is included in docs

#### What are the relevant tickets?
- [URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/186711902)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
